test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,50 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+});
```

Comment:
Dentro de la función de `describe` deben ir los `it` que tienes abajo. La razón de esto, es que un **describe** se utiliza como una definición de una suite (conjunto) de pruebas que tienen algo en común (probar la url https://httpbin.org/) y cada **it** corresponde a una prueba especifica.

Reply:
Le comentabamos a Yesid que en la página https://httpbin.org/ no había alguna prueba HEAD y nos sugirió eliminarla, no sé entonces cómo podría hacer ese tipo de test.

Reply:
El metodo HEAD consiste en solicitar los headers para cualquier metodo, en ese caso, pueden utilizar cualquier api existente, es decir, pueden hacer un head a: https://httpbin.org/get
Y verificar que el response contiene los headers